### ISSUE TITLE
Missing aria-required attribute

### ISSUE THREAD

**Issue by adamrparsons** [JedWatson/react-select#5006]
Hello, I'm currently implementing a form using React-Select and I've been told that the required attribute isn't coming through for screen-reader users. I've tried using MacOS's VoiceOver feature, and it seems that it reports it as "Combo box" along with the title, but doesn't mention required.

I saw that `aria-required` was listed in a large page of props [here](https://react-select.com/props#components) so I assumed it was part of the accepted props. Does React-Select support this attribute, or should I find another way to achieve this?

I've forked a codesandbox [here](https://codesandbox.io/s/react-select-v5-sandbox-y5jtm?file=/example.js), and as you can see when inspecting the dom, there are no aria-required attributes on any of the dom nodes

**Comment by imanpalsingh** [comment]
I'm picking this up

**Comment by Adamwojty** [comment]
Hi.
To add `aria-required="true"` attribute you can do something like this:

`import {components} from 'react-select'`
`<Select components={{Input: (props) => <components.Input {...props} aria-required="true" />`

Hope this helps
